src/overlays/actors/ovl_En_Toto/z_en_toto.c

```diff
@@ -205,8 +205,8 @@ void EnToto_Init(Actor* thisx, GlobalContext* globalCtx) {
     }
     ActorShape_Init(&this->actor.shape, 0.0f, func_800B3FC0, 30.0f);
     this->actor.bgCheckFlags |= 0x400;
-    SkelAnime_InitSV(globalCtx, &this->skelAnime, &D_0600A978, globalCtx->sceneNum == 0x12 ? &D_06003AA8 : &D_0600C880,
-                     this->limbDrawTbl, this->transitionDrawTbl, 18);
+    SkelAnime_InitFlex(globalCtx, &this->skelAnime, &D_0600A978,
+                       globalCtx->sceneNum == 0x12 ? &D_06003AA8 : &D_0600C880, this->jointTable, this->morphTable, 18);
```

Comment:
```suggestion
                       ( (globalCtx->sceneNum == 0x12) ? &D_06003AA8 : &D_0600C880 ), this->jointTable, this->morphTable, 18);
```
This is hard enough to read in the first place...